MadDonkeySoftware/corp-hq
# Move existing vue components to pages folder. Created demo component

### Story
https://maddonkeysoftware.myjetbrains.com/youtrack/issue/CHQ-94

### Summary
* Moved existing "components" contents to "pages" folder to more accurately reflect their purpose.
* Created example "component" to demonstrate how to include a component on one of the existing pages.
  * Extracted nav bar to be isolated component.
  * Created "link group" component to demonstrate re-use of component.

### Testing
* Run the UI as normal. 
* Browse to "#/Dashboard" to see the demo component in action.
